1-org/envs/shared/README.md

```diff
@@ -37,6 +37,20 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Run `terraform plan` and review output.
 1. Run `terraform apply`
 
+### Run tf-wrapper.sh script locally
+1. Change into 1-org folder.
+1. Run `cp ../build/tf-wrapper.sh .`
+1. Run `chmod 755 ./tf-wrapper.sh`
+1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Update backend.tf with your bucket from bootstrap. You can run
+```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```
+1. Run `./tf-wrapper.sh init shared`
```

Comment:
is it posible to use `prod` instead of `shared` here to make it more similar to the Cloud Build flow?

Reply:
Yes, changing already!